config/kustomization.yaml

```diff
@@ -6,5 +6,5 @@ commonLabels:
 bases:
 - crd
 - default
-#- webhook # Disable this if you're not using the webhook functionality.
+- webhook # Disable this if you're not using the webhook functionality.
```

Comment:
this kustomization file is missing the following:

```
patchesJson6902:
- target: # NOTE: This patch needs to be repeatd for EACH CustomResourceDefinition you have under crd/bases.
    group: apiextensions.k8s.io
    version: v1
    kind: CustomResourceDefinition
    name: haproxyloadbalancers.infrastructure.cluster.x-k8s.io
  path: patch_crd_webhook_namespace.yaml
- target: # NOTE: This patch needs to be repeatd for EACH CustomResourceDefinition you have under crd/bases.
    group: apiextensions.k8s.io
    version: v1
    kind: CustomResourceDefinition
    name: vsphereclusters.infrastructure.cluster.x-k8s.io
  path: patch_crd_webhook_namespace.yaml
- target: # NOTE: This patch needs to be repeatd for EACH CustomResourceDefinition you have under crd/bases.
    group: apiextensions.k8s.io
    version: v1
    kind: CustomResourceDefinition
    name: vspheremachines.infrastructure.cluster.x-k8s.io
  path: patch_crd_webhook_namespace.yaml
- target: # NOTE: This patch needs to be repeatd for EACH CustomResourceDefinition you have under crd/bases.
    group: apiextensions.k8s.io
    version: v1
    kind: CustomResourceDefinition
    name: vspheremachinetemplates.infrastructure.cluster.x-k8s.io
  path: patch_crd_webhook_namespace.yaml
- target: # NOTE: This patch needs to be repeatd for EACH CustomResourceDefinition you have under crd/bases.
    group: apiextensions.k8s.io
    version: v1
    kind: CustomResourceDefinition
    name: vspherevms.infrastructure.cluster.x-k8s.io
  path: patch_crd_webhook_namespace.yaml
```

this should patch the service namespace for the `clientConfig` using `patch_crd_webhook_namespace.yaml`